conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
```

Comment:
I left mode code related to testing from the neighborhoods repo in case we want to take this opportunity to add in some more testing infrastructure. @BrianHGrant I know this is a topic you have been thinking about and working on. That said, maybe those additions would be more appropriate in a future update.

Reply:
See my comment on api/tests.py, but I agree that now would be a good time to add in basic unit testing to the backend-exemplar. Actually I just implemented basic unit tess in the disaster-resilience repo for all of the endpoints (including your sandbox-related endpoints) and was just about to push them to the backend-exemplar.

Reply:
Looks good - I should probably rebuild my workstation to start checking this stuff out. I upgraded everything after Demo Day but since the Hack Oregon stuff is all in Docker that shouldn't matter.

Reply:
Looks good - I should probably rebuild my workstation to start checking this stuff out. I upgraded everything after Demo Day but since the Hack Oregon stuff is all in Docker that shouldn't matter.